src/TurnByTurn/RoutingSample.MAUI/MainPage.xaml.cs

```diff
@@ -1,46 +1,53 @@
-﻿using Esri.ArcGISRuntime.Security;
-using Esri.ArcGISRuntime.Xamarin.Forms;
+﻿
+
+using Esri.ArcGISRuntime.Security;
 using RoutingSample.ViewModels;
-using System;
-using System.ComponentModel;
-using System.Diagnostics;
-using Xamarin.Forms;
 
-namespace RoutingSample.Forms
+namespace RoutingSample.MAUI
 {
-    [DesignTimeVisible(false)]
     public partial class MainPage : ContentPage
     {
         private readonly MainViewModel _mainViewModel;
 
         public MainPage()
         {
+            // Create an API key at https://developers.arcgis.com/api-keys/, configure it with routing and geocoding services, then paste below.
+            Esri.ArcGISRuntime.ArcGISRuntimeEnvironment.ApiKey = "";
```

Comment:
Same here - not needed if we just sign in.
Also this should really be set in Program.cs as part of `UseArcGISRuntime()`
You can also use `#warning message here` to have compile time warnings generated reminding people to do something.